src/bucket/BucketManager.h

```diff
@@ -211,5 +214,19 @@ class BucketManager : NonMovableOrCopyable
     virtual void shutdown() = 0;
 
     virtual bool isShutdown() const = 0;
+
+    // Load the complete state of the ledger from the provided HAS. Throws if
+    // any of the buckets referenced in the HAS do not exist.
+    virtual std::map<LedgerKey, LedgerEntry>
```

Comment:
`std::map` or `std::unordered_map` ?

Reply:
oh nm, I see that you're using this as bucket order? So not a real bucket then (probably fine but some commentary would help)?

Reply:
Yes, this is to ensure they have a deterministic order. I'll make a note.